File: C/FirstNNaturalNumberReverse.c
#include<stdio.h>
#include<conio.h>
/* int main()
{
    int n,i;
    printf("Enter a number : ");
    scanf("%d",&n);
    for(i=1;i<=n;i++){
        printf("%d ",n+1-i);
    }
} */

int main()
{
    int n,i;
    printf("Enter a number : ");
    scanf("%d",&n);
    for(i=n;i>=1;i--)
        printf("%d ",i);
}

File: nitesh/java/FirstNNaturalNumberReverse.java
import java.util.Scanner;
public class FirstNNaturalNumberReverse
{
    public static void main(String[]args)
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number : ");
        int n = sc.nextInt();
        while(n>0){
            System.out.print(" "+n);
            n--;
        }
    }
}


